src/Hazelcast.Net/Core/TimeOut.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#nullable enable
+
+using System;
+
+namespace Hazelcast.Core
+{
+    /// <summary>
+    /// Provides time-out constants.
+    /// </summary>
+    public static partial class TimeOut
```

Comment:
Can we write it as `Timeout`  ?

Reply:
There's already a `Timeout` class in .NET so... no.

Reply:
class has been removed